pytorch_to_returnn/torch/nn/modules/module.py

```diff
@@ -773,12 +779,12 @@ def _base_get_output_shape_from_returnn(inputs_flat: List[Tensor], layer: LayerB
       if x.returnn_data.have_batch_axis():
         batch_size = input.shape[x.returnn_axis_from_torch_axis[x.returnn_data.batch_dim_axis]]
       for i in x.returnn_data.get_dynamic_axes():
-        dim_tag = x.returnn_data.get_dim_tag(i)
+        dim_tag, idx = _get_spatial_dim_tag_and_single_index(x.returnn_data, i)
```

Comment:
```suggestion
        dim_tag_ext = _get_spatial_dim_tag_and_single_index(x.returnn_data, i)
```
No need to separate them both here. That makes it also simpler when we need to change/extend `_get_spatial_dim_tag_and_single_index` at some point.